bin/test

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+rake parallel:test && rake parallel:features
```

Comment:
can you run both tasks from the same `rake` process? If it was

    rake parallel:test parallel:features

would it start them all in parallel or will it still wait for test to finish first?